README.md

```diff
@@ -196,3 +197,202 @@ be able to reproduce it. In particular, be sure to specify:
 
 `Trends.Earth` is free and open-source. It is licensed under the GNU General
 Public License, version 2.0 or later
+=======
```

Comment:
Can you check this? It looks like the file is duplicated - after this line the whole readme file repeats itself